.appveyor.yml

```diff
@@ -1,12 +1,13 @@
 install:
   - git submodule update --init
-  - set PATH=%PATH%;C:\cygwin\bin
   - python -m pip install --upgrade pip
   - pip install -r requirements.txt
   - pip install -r dev-requirements.txt
 
-build_script:
-  - make
+build: false
 
 test_script:
-  - make check
+  - tox
+
+on_success:
+  - tox -e codecov
```

Comment:
This does not seem to work, see: https://ci.appveyor.com/project/adobe-type-tools/psautohint/build/1.0.187#L193

Reply:
right, the command `codecov --env TOXENV` from tox.ini is failing with,

```
The filename, directory name, or volume label syntax is incorrect.
FIND: Parameter format not correct
```

I think **TOXENV** needs to be defined somewhere in the `.appveyor.yml` file.

Reply:
Still no luck. Looks Like it is using Unix `find` syntax with Windows tool with the same name. Now I wonder if this is meant to run on Windows at all…

Reply:
The changes may need to be more elaborate than https://github.com/adobe-type-tools/psautohint/pull/49/commits/d922b52c838e84f3ad4c6c03a5dacabf7727677e .
The fonttools configuration may provide some clues on what else is needed https://github.com/fonttools/fonttools/blob/master/.appveyor.yml

Reply:
On a closer look, it's working on the fonttools repo either https://ci.appveyor.com/project/fonttools/fonttools/build/1.0.1849/job/dyslxb2m2hl1vgah

And there are other people reporting the same problem,
https://github.com/codecov/codecov-node/issues/39#issuecomment-282522840
https://github.com/codecov/codecov-node/issues/2#issuecomment-287667265

Reply:
That message (`FIND: Parameter format not correct`) happens even after hardcoding the file to upload, but later on the report says,
```
  codecov: commands succeeded
  congratulations :)
```
Using just `codecov --env TOXENV` produces the same results, so I think you can nuke the last commit.

Reply:
@khaledhosny you should definitely remove my last commit https://github.com/adobe-type-tools/psautohint/pull/49/commits/c15c7c9faffe1edc6442a1a37176026964af867d because it shows up as an error at https://codecov.io/gh/adobe-type-tools/psautohint/commits
 
And [this page](https://codecov.io/gh/adobe-type-tools/psautohint/commit/6d103e8b45b8eb09d446e0f1493ef3e21792bcfd/build) confirms that Codecov received all the uploads from your last commit https://github.com/adobe-type-tools/psautohint/pull/49/commits/6d103e8b45b8eb09d446e0f1493ef3e21792bcfd

Reply:
OK, removed. I think we can merge now.

Reply:
👍